internal/oem/oem.go

```diff
@@ -103,10 +105,129 @@ func init() {
 		provider: gce.Creator{},
 		config: types.Config{
 			Systemd: types.Systemd{
-				Units: []types.SystemdUnit{{
-					Name:   "coreos-metadata-sshkeys@.service",
-					Enable: true,
-				}},
+				Units: []types.SystemdUnit{
+					{
+						Name:   "coreos-metadata-sshkeys@.service",
+						Enable: true,
+					},
+					{
+						Name:   "google-accounts-manager.service",
+						Enable: true,
+						Contents: `
+[Unit]
```

Comment:
Adding the ability to copy or symlink sounds reasonable, after all even
sysyemctl even has support for linking things into the search path like
that. :)
On May 11, 2016 9:28 AM, "Alex Crawford" <notifications@github.com> wrote:

> In internal/oem/oem.go
> <https://github.com/coreos/ignition/pull/236#discussion_r62880513>:
>
> > @@ -103,10 +105,129 @@ func init() {
> >  		provider: gce.Creator{},
> >  		config: types.Config{
> >  			Systemd: types.Systemd{
> > -				Units: []types.SystemdUnit{{
> > -					Name:   "coreos-metadata-sshkeys@.service",
> > -					Enable: true,
> > -				}},
> > +				Units: []types.SystemdUnit{
> > +					{
> > +						Name:   "coreos-metadata-sshkeys@.service",
> > +						Enable: true,
> > +					},
> > +					{
> > +						Name:   "google-accounts-manager.service",
> > +						Enable: true,
> > +						Contents: `
> > +[Unit]
>
> I'm not sure how this would work. If the units were installed to
> /usr/share/oem, how would we enable them? Right now, Ignition doesn't have
> the ability to copy files (we could add it though).
>
> While it is unfortunate that this info in scattered across repos, I'm not
> sure it's the worst thing. Ignition already knows a lot about the specific
> platform, and this is just a matter of moving the configs completely from
> coreos-overlay to Ignition. It's still a little split with GCE, but I plan
> on cleaning that up before this ships.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/ignition/pull/236/files/5a32091b9076a90df244e0a193debf71505f4cb6#r62880513>
>